content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -283,52 +233,42 @@ node {
     /* .. snip .. */
 }
 ----
-<1> Accessing the `currentBuild.result` variable allows the Pipeline to
-determine if there were any test failures. In which case, the value would be
-`UNSTABLE`.
+<1> Accessing the `currentBuild.result` variable allows the Pipeline to determine if there were any test failures.
+In which case, the value would be `UNSTABLE`.
 
-Assuming everything has executed successfully in the example Jenkins Pipeline,
-each successful Pipeline run will have associated build artifacts archived,
-test results reported upon and the full console output all in Jenkins.
+Assuming everything has executed successfully in the example Jenkins Pipeline, each successful Pipeline run will have associated build artifacts archived, test results reported upon and the full console output all in Jenkins.
 
 
-A Scripted Pipeline can include conditional tests (shown above), loops,
-try/catch/finally blocks and even functions. The next section will cover this
-advanced Scripted Pipeline syntax in more detail.
+A Scripted Pipeline can include conditional tests (shown above), loops, try/catch/finally blocks and even functions.
```

```suggestion
A Scripted Pipeline can include conditional tests (shown above), loops, try/catch/finally blocks, and even functions.
```
Suggestion to add punctuation for Oxford comma.